sources/packages/forms/src/form-definitions/studentadditionaltransportationappeal.json

```diff
@@ -143,6 +143,72 @@
               "properties": {},
               "lockKey": true
             },
+            {
+              "autofocus": false,
+              "input": true,
+              "tableView": true,
+              "inputType": "radio",
+              "label": "Are you listed as a driver on the insurance for the vehicle you will be using?",
```

Comment:
I believe `onlyAvailableItems:true` should be there for appeal forms as well ? Is there a reason for that nbot being here?